lib/pdn/Kconfig

```diff
@@ -7,11 +7,15 @@
 menuconfig PDN
 	bool "Packet Domain Network library"
 	depends on NRF_MODEM_LIB && AT_MONITOR
-	# library uses heap
-	depends on HEAP_MEM_POOL_SIZE > 0
 
 if PDN
 
+config HEAP_MEM_POOL_ADD_SIZE_PDN
+	int
+	default 64
```

Comment:
Since `sizeof(struct pdn)` is 12, should we make this default a multiple of 12?

Reply:
Not really, allocations have their own metadata overhead on the heap, which is why i didn't make the default 12 for the default context only.